RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -250,6 +250,46 @@ boolean LX_islandAccess()
 	return false;
 }
 
+boolean LX_lockPicking()
+{
+	if(!auto_have_skill($skill[Lock Picking]))
+	{
+		return false;
+	}
+
+	if(get_property("lockPicked").to_boolean() == true)

```

Comment:
```suggestion
	if(get_property("lockPicked").to_boolean())
```
to_boolean() converts it into a true or false.
so it is either if(true) or if(false), there is no need to do if(true == true)

Reply:
also if you want the opposite then you use the format of
`if(!get_property("PropertyName").to_boolean())`